CHANGELOG/7.0.md

```diff
@@ -1,5 +1,47 @@
 # 7.0 Changelog
 
+## [7.0.1] - 2020-05-14
```

Comment:
Are we targeting tomorrow for publishing the release? If so, this should be changed to 05-15 I guess

Reply:
It's releasing now.   This is not a review... Just a merge...